### ISSUE TITLE
Building any examples for esp32/m5paper fails

### ISSUE THREAD

**Issue by FWeinb** [Moddable-OpenSource/moddable#821]
**Build environment:** macOS
**Target device:** esp32/m5paper
**Description**
When trying to build any of the example  for the esp32/m5paper the build fails with 
`### TypeError: cannot configure property!`

**Steps to Reproduce**
1. Build the epaper-flashcards example using: `mcconfig -v -m -p esp32/m5paper`
3. Get error: `### TypeError: cannot configure property!`

**My Investigation**
1. Redo the `xsl` build-part from the verbose output of mcconfig using the debug build 
2. `xsl [...]/moddable/build/bin/mac/debug/xsl -b [...]/moddable/build/tmp/esp32/m5paper/debug/epaper-flashcards/modules [...]`
3.  I get the error:
```
[...]/moddable/modules/drivers/it8951/it8951.js:350: exception: (host): cannot configure property!
```

The issue seams to be this line:
https://github.com/Moddable-OpenSource/moddable/blob/e55fbbf8c9a450acb052fc1630171f04a11c25c2/modules/drivers/it8951/it8951.js#L350 

Reason seams to be that `Object.freeze()` on `TypedArrays` isn't working as I can reproduce this issue using `xst -e "Object.freeze({ obj: Uint8Array.of(1,2,3) }, true)"` causing the same error.

**Comment by phoddie** [comment]
Hey, @FWeinb! Good to hear from you. Thank you for the report. We'll take a look. Until then, you can safely run by commenting out that `Object.freeze` call. It will cost you a few bytes of RAM, but otherwise should be the same.

**Comment by phoddie** [comment]
FYI – the root cause here is related to some changes we made recently to the way deep freeze is implemented for objects. The details get into ECMAScript conformance. But... the call to `Object.freeze` shouldn't throw in this case, as it didn't before. We've fixed that and are scheduled to push that to this repository Monday. Thanks again for the report. The concise `xst` example made it easy to track down.
